react/src/Toggle/Toggle.tsx

```diff
@@ -19,7 +19,7 @@ export interface ToggleProps extends SwitchProps {
   /**
    * Secondary description text
    */
-  description?: string
+  description?: string | ReactNode
```

Comment:
nit: ReactNode consists of `string`

```suggestion
  description?: ReactNode
```

Reply:
lol my bad, thanks for flagging! fixed